tests/test_defects.py

```diff
@@ -86,14 +89,16 @@ def setUp(self):
 
         # NOTE: darks and flats have same defects applied deliberately to both
         for defect in self.brightDefects:
-            y, x, s = defect
-            flatData[x:x+s, y:y+s] += self.nSigmaBright * flatWidth  # are these actually the numbers we want?
-            darkData[x:x+s, y:y+s] += self.nSigmaBright * darkWidth  # are these actually the numbers we want?
+            y, x, sy, sx = defect
+            # are these actually the numbers we want?
```

Comment:
I don't know what the meaning of this question is. I believe it was Merlin who wrote it.

Reply:
`git blame` says it wasn't me.

Reply:
`git blame` is wrong. :-) Try [here](https://github.com/lsst/cp_pipe/commit/0712f5971e724ea276a0e532723169cdb135ef69#diff-24c51ebe1c07d9827b3a5873b938d54bR90).

Reply:
_You_, sir, are the one who is wrong - I put them on a totally different line in the commit you reference! The fact that someone (who git correctly blames) moved them so that they now refer to entirely different code (my code that it referred to having been deleted) means that I am indeed _not_ to blame. So there! 🎩

Reply:
Anyway, shenanigans and 💩-posting aside, I think these are the ~droids~ numbers you're looking for. They'll certainly do for now, anyway.

Reply:
So feel free to delete that comment.